src/main/java/command/stats/StatsHelpCommand.java

```diff
@@ -0,0 +1,17 @@
+package command.stats;
+
+public class StatsHelpCommand implements StatsCommand {
+    /**
+     * Prints the help message for the stats commands
+     */
+    public static void execute() {
+        System.out.println("Available commands:");
+        System.out.println("\tbestselling - View the best selling item(s)");
+        System.out.println("\ttotal orders - View the total number of orders");
+        System.out.println("\trevenue -gross - View the gross revenue");
+        System.out.println("\trevenue -net - View the net revenue");
+        System.out.println("\tview profit -cost <cost> - View the profit based on the cost");
```

Comment:
What does profit based on the cost mean? Doesn't gross revenue reflect revenue without costs, which is the profit?

Reply:
Upon some research I discovered that net revenue is not the same as profit, but you could clarify the command either in the user guide or above so it is clearer for users without the necessary knowledge

Reply:
there are more detailed explanations of profit and revenue in the UG; can add on if they are not precise enough